carmin-server/server/resources/helpers/path.py

```diff
@@ -111,3 +118,22 @@ def parent_dir_exists(requested_data_path: str) -> bool:
     parent_directory = os.path.abspath(
         os.path.join(requested_data_path, os.pardir))
     return os.path.exists(parent_directory)
+
+
+def platform_path_exists(url_root: str, platform_path: str) -> (bool, str):
+    path_url = '{}path/'.format(url_root)
+
+    if isinstance(platform_path, list):
+        for path in platform_path:
+            exists = os.path.exists(
+                os.path.join(app.config['DATA_DIRECTORY'],
+                             path[len(path_url):]))
+            if not exists:
+                return exists, path
```

Comment:
I would maybe change to return False, path, just to make it more visually explicit that this is a fail result.